ui/imports/shared/status/StatusGifPopup.qml

```diff
@@ -70,9 +70,11 @@ Popup {
         } else {
             confirmationPopup.open()
         }
+        gifsLoader.active = true
     }
 
     onClosed: {
+        gifsLoader.active = false
```

Comment:
you can add binding `active: popup.opened`

Reply:
@borismelnik yes, it could be like this or like that, not a big deal.